help/include/inbox-instructions.md

```diff
@@ -8,32 +8,22 @@
    your **inbox** when done (e.g., by using the **back** button in your browser
    or desktop app) to select the next conversation.
 
-!!! tip ""
-
-    You can collapse or expand the list of topics in a channel by clicking the
-    **collapse**
-    (<i class="zulip-icon zulip-icon-arrow-down"></i>)
-    or **expand**
-    (<i class="zulip-icon zulip-icon-arrow-down icon-collapsed-state"></i>)
-    icon to the left of a channel name.
-
 {tab|mobile}
 
-1. Tap the **Inbox**
-   (<img src="/static/images/help/mobile-inbox-icon.svg" alt="inbox" class="help-center-icon"/>)
+1. Tap the **Inbox** (<i class="zulip-icon zulip-icon-inbox mobile-help"></i>)
    tab in the bottom left corner of the app.
 
 2. Tap on a conversation you're interested in to view it. You can return to
-   **Inbox** when done (e.g., by using the **back* button) to select the next
-   conversation.
+   your **inbox**  when done (e.g., by using the **back** button) to select
+   the next conversation.
+
+{end_tabs}
 
 !!! tip ""
 
-    You can collapse or expand the list of topics in a channel by tapping the
+    You can collapse or expand the list of topics in a channel by clicking the
```

Comment:
Hm, why tapping -> clicking here?

Reply:
Since the icons for collapsing and expanding these are the same in the web/desktop and mobile apps now, I pulled the tip out of the tabbed instructions to be a general tip. But there is that verb difference of tap/click between how we write the documentation for mobile vs web/desktop.

![Screenshot from 2025-06-11 12-05-11](https://github.com/user-attachments/assets/65e78d8f-b49c-425a-876f-1d1f90b8f581)